### DISCUSSION TITLE
Separate Front and Back End?

### DISCUSSION THREAD

**Discussion by voskuil23** [frangoteam/FUXA#1004] [Q&A] [answered]
I am attempting  to run Fuxa in a network with very strict rules.
I have 2 Servers:
-Server 1  - Communicates with the Field Devices and Server 2
-Server 2 - Communicates on Intranet and with Server 1

I want to host the Backend of Fuxa on Server 1, where it polls the field devices via modbus and stores data in influxdb. In Server 2, I simply want to host the webpage so that it is accessible on the network from all other clients. Can a separate Fuxa instance connect to another one at a different IP?

I know the typical install is that Fuxa runs on one machine, but in my case that won't be possible. Is there some work around to allow what Im suggesting above?

![network1](https://github.com/frangoteam/FUXA/assets/152212501/fd498ab8-a035-425b-9c65-dbcec7cc6196)

Thanks in advance!

**Comment by unocelli** [comment, marked as answer]
Hi,
currently you cannot do that, the communication between backends is missing.
